src/commons/constant/thailand-address/district.json

```diff
@@ -1 +1,7426 @@
\ No newline at end of file
+[
```

Comment:
Do we really need this file since we already has district.ts

Reply:
Poom forget to delete